homeassistant/components/hassio.py

```diff
@@ -261,7 +333,7 @@ def send_command(self, command, method="post", payload=None, timeout=10):
                         X_HASSIO: os.environ.get('HASSIO_TOKEN')
                     })
 
-                if request.status != 200:
+                if request.status not in (200, 400):
```

Comment:
`400` indicates bad request. That's an error status, no?

Reply:
400 and 200 have a API json payload

Reply:
ok